addons/base_import_module/static/src/base_import_list_renderer.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module */
+import { useService } from "@web/core/utils/hooks";
+import { ListRenderer } from "@web/views/list/list_renderer";
+
+export class ImportListRenderer extends ListRenderer {
+
+    setup() {
+        super.setup()
+        this.action = useService("action");
+    }
+
+    get hasSelectors() {
+        return super.hasSelectors && this.props.list.records[0]?._values.module_type != 'industries';
```

Comment:
it seems we only check the first record, shouldn't check if there is (any)? (if we have a mix of module_types in the list

Reply:
The problem is only in the 'Industries' tab, where no button works (activate, upgrade, print, ...). So first record is enough to disable the selector.

Industries also appears in 'All' tab when installed, I agree, but in this case, I don't think it is valuable to remove the selector just because an industry is installed. Furthermore, there is no bug anymore with the button if the industry is installed: the button either doesn't do anything (activate/upgrade), either works normally (print).

So I'd say this is sufficient. Or we should check if there is any unistalled industry in the view maybe.

Reply:
Even if it is installed, you should not display the selectors, as it'll let you call the standard immediate_install instead of imediate_install_apps which is not possible on the kanban view and form view.

So 'Upgrade' won't do the same thing in tree view than in the other views

Reply:
Agreed!